### PR TITLE
add my card

### PR DESCRIPTION
**Pull request by newzpanuwat**
add my profile

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2562,7 +2562,33 @@ <h2>A project for first-time contributions</h2>
           </div>
         </div>
 
-        <!-- ________ Alina's card END _____ -->
+        <!-- ________ New Panuwat card END _____ -->
+
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```
Comment (TenTraicion) [inline index.html:2567]:
```suggestion
        <!-- ________ *New Panuwat* Contributor card START ________  -->
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@newzpanuwat, Thanks for the contribution. Kindly , move your card to the top of this section where it says `Paste your card directly below this line` after line 117. Thanks!
and also make the change mentioned below.

for any kind of query you can ask here or in our [Discord Community: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF)

File: index.html [hunk 2]
```diff
@@ -2562,7 +2562,33 @@ <h2>A project for first-time contributions</h2>
           </div>
         </div>
 
-        <!-- ________ Alina's card END _____ -->
+        <!-- ________ New Panuwat card END _____ -->
```
Comment (TenTraicion) [inline index.html:2565]: Don't edit others card.
```suggestion
        <!-- ________ Alina's card END _____ -->
```

File: index.html (hunk 1 above)
Comment (newzpanuwat) [reply index.html:2567]: Okay. Thank you so much and sorry i made mistake this line of code.

**Comment by CandidDeer** [comment]
Thank you for your contribution and for making the requested changes. I have merged your PR and your card will be live in a moment.

**Pull request merged**